index.html

```diff
@@ -18,20 +19,142 @@
+            </div>
+          </div>
+          <button class="see-project secondary-button">See Project</button>
+        </section>
+        <section class="card second-card">
+          <div class="mask-group">
+            <h4>Profesional Art Printing Data</h4>
+            <p class="supporting-text-secondary-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="tag-holder">
+              <div class="secondary-tag"><p class="html">html</p></div>
+              <div class="secondary-tag"><p class="bootstrap">bootstrap</p></div>
+              <div class="secondary-tag"><p class="rubi">Ruby</p></div>
+            </div>
+          </div>
+          <button class="see-project secondary-button">See Project</button>
+        </section>
+        <section class="card second-card">
+          <div class="mask-group">
+            <h4>Profesional Art Printing Data</h4>
+            <p class="supporting-text-secondary-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="tag-holder">
+              <div class="secondary-tag"><p class="html">html</p></div>
+              <div class="secondary-tag"><p class="bootstrap">bootstrap</p></div>
+              <div class="secondary-tag"><p class="rubi">Ruby</p></div>
+            </div>
+          </div>
+          <button class="see-project secondary-button">See Project</button>
+        </section>
+        <section class="card second-card">
+          <div class="mask-group">
+            <h4>Profesional Art Printing Data</h4>
+            <p class="supporting-text-secondary-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="tag-holder">
+              <div class="secondary-tag"><p class="html">html</p></div>
+              <div class="secondary-tag"><p class="bootstrap">bootstrap</p></div>
+              <div class="secondary-tag"><p class="rubi">Ruby</p></div>
+            </div>
+          </div>
+          <button class="see-project secondary-button">See Project</button>
+        </section>
+    </section>
+    <section class="about-section">
+      <section class="about">
+        <h2 class="about-title">About me</h2>
+        <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div class="about-button-container"><button class="see-project about-button">Get My Resume</button></div>
+      </section>
+      <section class="skill">
+        <div class="white-vector"></div>
+        <ul class="skill-list-container">
+          <li class="slk">Language</li>
```

Comment:
- The list of icons uses a `ul` tag to group them, and `li` per each item.

Reply:
## **Hi @carlos-ssh** 
**Thanks for your review!** 👍 
Could you elaborate on this change, please? I Believe I have those Items grouping the skills (Language, Frameworks, Skills). The only element outside of this ul tag is the vector (I Put it inside this section)

Reply:
...before the **UL** list

Reply:
Hi @carloshs1994,
The list of technology in each category(e.g Javascript, Html, CSS) is a list and should be grouped under an unordered list `<ul>` tag. They should be grouped under their respective categories. E.g (Javascript, Html, CSS, Ruby) should be grouped under the Languages category, (Bootstrap, Ruby on Rails,...) should be grouped under the Frameworks category.

Reply:
## Thanks @keneogu !
That was helpful 🥇